index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <title>Daniel Hopton</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+        <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
+    </head>
+    <body>
+        <nav>
+            <div id="navwrapper">
```

Comment:
exceeds the horizontal viewport at 320px (so the user has to scroll horizontally to see the full width of the nav bar).  It's generally good design to ensure there is no horizontal scrolling at 320px.